jsk_2016_01_baxter_apc/euslisp/main.l

```diff
@@ -70,7 +70,7 @@
               (setq past-graspingps nil)
               (setq target-bin (str2symbol (send order :bin)))
               (setq target-obj (send order :object))
-              (setq bin-contents (send *ri* :get-bin-contents target-bin))
+              (setq bin-contents (get-bin-contents target-bin))
```

Comment:
why not `(ros::get-param (format nil "/bin_contents/bin_~a" (symbol2str target-bin))`?